mvpa2/tests/test_clf.py

```diff
@@ -335,7 +335,8 @@ def test_degenerate_usage(self, clf):
 
 
     # TODO: sg - remove our limitations, meta, lda, qda and skl -- also
-    @sweepargs(clf=clfswh['!sg', '!plr', '!meta', '!lda', '!qda', '!glmnet'])
+    @sweepargs(clf=clfswh['one-class', '!sg', '!plr', '!meta', '!glmnet'] + \
+                   clfswh['one-class-binary', '!sg', '!plr', '!meta', '!glmnet'])
```

Comment:
I guess with introduction of those tags it could simply be `['oneclass', 'oneclass-binary']` without any ugly exclusions? (may be  with '!meta' at most)?